resources/views/home.blade.php

```diff
@@ -10,6 +10,16 @@
 
 
     <div class="main py-4">
+        @can("admin.versioncheck.read")
+            @if(config("app.version") < config("LATESTVERSION"))
```

Comment:
check if the storage version file exists, if it does. use that to compare the versions